pkg/operators/cli/clioperator.go

```diff
@@ -179,6 +180,36 @@ func (o *cliOperatorInstance) ExtraParams(gadgetCtx operators.GadgetContext) api
 	return api.Params{fields, mode}
 }
 
+func parseFields(fieldsString string, defaultFields []string) []string {
+    fields := strings.Split(fieldsString, ",")
+    result := make([]string, len(defaultFields))
```

Comment:
```suggestion
    result := make([]string, 0, len(defaultFields))
```

(otherwise the result will contain a lot of empty elements and fail)